lib/manageiq/ui/classic/engine.rb

```diff
@@ -1,7 +1,16 @@
 require 'rails/all'
-require 'sass-rails'
-require 'coffee-rails'
-require 'patternfly-sass'
+
+if ENV["RAILS_ENV"] != "production" || defined?(Rake)
+  require 'sass-rails'
+  require 'coffee-rails'
+  require 'patternfly-sass'
+else
+  require 'bootstrap-sass/engine'
+  require 'font_awesome/sass/rails/engine'
```

Comment:
@himdel The nice thing about this is that it these changes will do almost nothing until the `patternfly-sass` change is merged and used. (meaning sass will still be loaded until we do as well), so the app should order independent of those changes.

I think the webpack stuff is the better long term goal, and this is just some quick wins to try and reduce our memory foot print that we might try and get into other releases (fine if possible, euwe maybe...), where I see the webpack stuff just being something we go with moving forward).

I will put some quick steps in the PR description so you guys can check to see if this is causing issues or something isn't working on the javascript/css end that I am not aware of.